src/api/subscriptions.js

```diff
@@ -375,6 +386,8 @@ export let SaveSubscriptionRequestCallback;
  *
  * @typedef {{
  *   theme: string,
+ *   lang: (string|undefined),
+ *   isClosable: (boolean|undefined),
```

Comment:
`isClosable` will be passed to the `showOffers` API call when `dialog` is opened.